modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/AccountDetailsViewModel.kt

```diff
@@ -72,10 +73,9 @@ class AccountDetailsViewModel
     val accountStartDate: LiveData<Date> = MutableLiveData<Date>().apply { value = Date(statsManager.statsStartTime) }
 
     val marketingOptInState: LiveData<Boolean> =
-        settings.marketingOptObservable
-            .distinctUntilChanged()
```

Comment:
Because this is a state flow now, we don't need `distinctUntilChanged` anymore.

Reply:
Nice! Thanks for pointing it out. 🙇‍♀️